protocol.md

```diff
@@ -189,9 +196,20 @@ With the possible values for the format of the transaction used for signing bein
 
 With the possible values for the format of the signature itself being 0x00 for DER format, and 0x01 for (R,S) format.
 
-#### CHECK_ASSET_IN_LEGACY
+#### CHECK_ASSET_IN_LEGACY_AND_DISPLAY
+
+This command is DEPRECATED. \
+Please use CHECK_ASSET_IN_AND_DISPLAY instead. The data content is the same, only the INS byte is different (and does not collide with CHECK_PAYOUT_ADDRESS) \
```

Comment:
change to "Please use CHECK_ASSET_NO_DISPLAY"

Reply:
Redone the whole paragraph